### PR TITLE
Add some components to Strapi [NGC-1219]

### PR DESCRIPTION
**Pull request by bjlaa**

### CODE REVIEW

File: .tmp/data.db
[the remark this answers is not in the document]
Comment (bjlaa) [reply .tmp/data.db:1]: ✅

File: package.json [hunk 1]
```diff
@@ -12,8 +12,10 @@
     "start": "strapi start"
   },
   "dependencies": {
+    "@strapi/plugin-seo": "^2.0.1-rc.1",
     "@strapi/plugin-users-permissions": "5.0.6",
     "@strapi/strapi": "^5.0.6",
+    "better-sqlite3": "^11.3.0",
```
[the remark this answers is not in the document]
Comment (bjlaa) [reply package.json:18]: Dunno, I added some plugins via the Strapi UI, did not make this conscious choice 😄

File: public/uploads/Capture_d_ecran_2024_10_10_a_13_21_14_3f34789b02.png
[the remark this answers is not in the document]
Comment (bjlaa) [reply public/uploads/Capture_d_ecran_2024_10_10_a_13_21_14_3f34789b02.png:1]: ✅

File: src/api/article/content-types/article/schema.json [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+      },
+      "component": "element.faq"
+    },
+    "categories": {
+      "type": "relation",
+      "relation": "manyToMany",
+      "target": "api::category.category",
+      "inversedBy": "articles"
+    },
+    "cover_image": {
```
[the remark this answers is not in the document]
Comment (bjlaa) [reply src/api/article/content-types/article/schema.json:60]: Right !

File: src/api/article/content-types/article/schema.json [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+      "type": "blocks",
+      "required": true
+    },
+    "content": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "richtext"
```
Comment (bjlaa) [inline src/api/article/content-types/article/schema.json:45]: `richtext` ici c'est voulu ?

File: src/api/article/content-types/article/schema.json
[the remark this answers is not in the document]
Comment (paulsouche) [reply src/api/article/content-types/article/schema.json:1]: On slugify le titre ?

**Review by paulsouche — APPROVED** [review]

**Pull request merged**